nextflow_schema.json

```diff
@@ -23,12 +23,34 @@
                     "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/phaseimpute/usage#samplesheet-input).",
                     "fa_icon": "fas fa-file-csv"
                 },
+                "input_region": {
+                    "type": "string",
+                    "description": "Region of the genome to use (optional: if no file given, the whole genome will be used). The file should be a comma-separated file with 3 columns, and a header row.",
+                    "schema": "assets/schema_input_region.json",
+                    "format": "file-path",
+                    "pattern": "^\\S+\\.csv$"
+                },
                 "outdir": {
                     "type": "string",
                     "format": "directory-path",
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "rename_chr": {
+                    "type": "boolean",
+                    "description": "Should the panel vcf files be renamed to match the reference genome (e.g. 'chr1' -> '1')",
+                    "pattern": "true|false"
+                },
+                "max_chr_names": {
+                    "type": "integer",
+                    "description": "Maximum number of contigs name to print before resuming (i.e. show only subset and add '...' at the end).",
+                    "hidden": true,
+                    "default": 4
+                },
+                "remove_samples": {
+                    "type": "string",
+                    "description": "Comma-separated list of samples to remove from the reference panel. Useful for benchmarking purposes."
```

Comment:
add a pattern checking for a comma seperated list

Reply:
May I ask what you mean here? The idea is that if the user has a a reference haplotype panel that includes samples that he/she might want to remove, he could specify the names of the samples here. There is no pattern of those sample names.

Reply:
ohhh I see, very nice approach, I'll add that. thank you!